API.md

```diff
@@ -115,18 +115,19 @@ Onyx.disconnect(connectionID);
 ```
 <a name="notifySubscribersOnNextTick"></a>
 
-## notifySubscribersOnNextTick(key, value)
+## notifySubscribersOnNextTick(key, value, [canUpdateSubscriber])
 This method mostly exists for historical reasons as this library was initially designed without a memory cache and one was added later.
 For this reason, Onyx works more similar to what you might expect from a native AsyncStorage with reads, writes, etc all becoming
 available async. Since we have code in our main applications that might expect things to work this way it's not safe to change this
 behavior just yet.
 
 **Kind**: global function
 
-| Param | Type |
-| --- | --- |
-| key | <code>String</code> |
-| value | <code>\*</code> |
+| Param | Type | Description |
+| --- | --- | --- |
+| key | <code>String</code> |  |
+| value | <code>\*</code> |  |
+| [canUpdateSubscriber] | <code>function</code> | only subscribers that pass this truth test will be updated |
```

Comment:
Maybe add a better description for this or an example, to make clearer how to use it.